src/main/java/com/sap/oss/phosphor/fosstars/data/github/SecuregoDataProvider.java

```diff
@@ -0,0 +1,263 @@
+package com.sap.oss.phosphor.fosstars.data.github;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.RUNS_SECUREGO_SCANS;
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.USES_SECUREGO_SCAN_CHECKS;
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.USES_SECUREGO_WITH_RULES;
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.setOf;
+
+import com.sap.oss.phosphor.fosstars.data.AbstractStaticScanToolsDataProvider;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.ValueSet;
+import com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.GitHubProject;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.oss.phosphor.fosstars.util.Yaml;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.regex.Pattern;
+import org.apache.commons.collections4.IteratorUtils;
+
+/**
+ * The data provider gathers info about how a project uses Bandit for static analysis. In
+ * particular, it tries to fill out the following features:
+ *
+ * <ul>
+ *   <li>{@link OssFeatures#USES_SECUREGO_SCAN_CHECKS}
+ *   <li>{@link OssFeatures#RUNS_SECUREGO_SCANS}
+ *   <li>{@link OssFeatures#USES_SECUREGO_SCAN_CHECKS}
+ * </ul>
+ */
+public class SecuregoDataProvider extends AbstractStaticScanToolsDataProvider {
+
+  /** A step in a GitHub action that triggers analysis with Securego. */
```

Comment:
```suggestion
  /** 
    * A step in a GitHub action that triggers analysis with Securego.
    */
```
Please follow the same everywhere else.